examples/tutorials/URDF_robotics_tutorial.py

```diff
@@ -145,9 +139,9 @@ def test_urdf_memory():
     robot_file = urdf_files[robot_key]
     for _sample in range(1000):
 
-        robot_id = sim.add_articulated_object_from_urdf(robot_file)
+        robot = art_obj_mgr.add_articulated_object_from_urdf(robot_file)
         process_memory_tracking.append(get_process_memory_usage())
-        sim.remove_articulated_object(robot_id)
+        sim.remove_articulated_object(robot.object_id)
```

Comment:
This should be accomplished via the articulated object manager.

Reply:
We should considering adding a Python deprecation warning to these calls.